frontend/src/components/homepage/sections/HomepageExperimental.tsx

```diff
@@ -0,0 +1,49 @@
+import { UswdsIconNames } from "src/types/generalTypes";
+
+import { useMessages, useTranslations } from "next-intl";
+import { Button, Grid } from "@trussworks/react-uswds";
+
+import HomePageSection from "src/components/homepage/homePageSection";
+import IconInfo from "src/components/homepage/IconInfoSection";
+
+const trySearchLink = "https://simpler.grants.gov/search";
+
+const ExperimentalContent = () => {
+  const t = useTranslations("Homepage.sections.experimental");
+  const messages = useMessages() as unknown as IntlMessages;
+  const { iconSections } = messages.Homepage.sections.experimental;
+
+  return (
+    <HomePageSection className="bg-base-lightest" title={t("title")}>
+      <h2 data-testid="homepage-experimental">{t("canDoHeader")}</h2>
+      <h3>{t("canDoSubHeader")}</h3>
+      <p className="font-sans-md line-height-sans-4">{t("canDoParagraph")}</p>
+      <a href={trySearchLink} target="_blank" rel="noopener noreferrer">
```

Comment:
do we really want this to open in a new tab? I assume that getting users onto search is one of the main goals of this page, so it seems like we wouldn't want to make them move to a new tab